src/app/post-verification/invited-user/invited-user.page.ts

```
@@ -129,4 +117,28 @@ export class InvitedUserPage implements OnInit {
       this.trackingService.showToastMessage({ ToastContent: message });
     }
   }
+
+  checkPasswordEquality(): void {
+    if (!this.fg || !this.fg.controls) {
+      return null;
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)